Adyen/UI/Form/Items/Text/FormTextInputItem.swift

```diff
@@ -8,8 +8,10 @@ import Foundation
 
 /// An item for plain text input
 /// :nodoc:
-public final class FormTextInputItem: FormTextItem {
+public class FormTextInputItem: FormTextItem {
```

Comment:
this supposed to be a final class. Use `FormTextItem ` if you need to override something

Reply:
made it final